libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java

```diff
@@ -84,12 +90,15 @@ static void departEvent(SessionState sessionState, RouteProgress routeProgress,
       (int) routeProgress.distanceTraveled(), (int) routeProgress.distanceRemaining(),
       (int) routeProgress.durationRemaining(), sessionState.startTimestamp()
     );
-    MapboxTelemetry.getInstance().addAbsoluteDistanceToDestination(0, departEvent);
+
+    calculateAbsoluteDistance(location, routeProgress);
```

Comment:
Not completely sure why we're calling `calculateAbsoluteDistance(location, routeProgress);` here.
We're calling it again 👇

Reply:
What about extracting it to a local variable (e.g. `absoluteDistance`) instead of passing the call into `addAbsoluteDistanceToDestination()`?